staging/src/k8s.io/component-base/cli/flag/ciphersuites_flag.go

```diff
@@ -23,39 +23,24 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
-// ciphers maps strings into tls package cipher constants in
-// https://golang.org/pkg/crypto/tls/#pkg-constants
-// to be replaced by tls.CipherSuites() when the project migrates to go1.14.
-var ciphers = map[string]uint16{
```

Comment:
add a test case to TestStrToUInt16 that specifies all of the existing secure and insecure strings, ensure it passes on master and on this PR to make sure we're not dropping/changing any mappings